crypto-ffi/Makefile.toml

```diff
@@ -156,13 +156,23 @@ script = '''
     mv ./bindings/jvm/src/main/kotlin/com/wire/crypto/core_crypto_ffi.kt ./bindings/jvm/src/main/kotlin/com/wire/crypto/CoreCrypto.kt
 
     sed -i.bak \
-        -e 's/\CryptoException\b/CryptoError/g' \
+        -e 's/\bCryptoException\b/CryptoError/g' \
         -e 's/\bE2eIdentityException\b/E2eIdentityError/g' \
         ./bindings/jvm/src/main/kotlin/uniffi/core_crypto/core_crypto.kt
 
     rm ./bindings/jvm/src/main/kotlin/uniffi/core_crypto/core_crypto.kt.bak
 '''
 
+[tasks.ffi-kotlin.mac]
+script = '''
+    mv ./bindings/jvm/src/main/kotlin/com/wire/crypto/core_crypto_ffi.kt ./bindings/jvm/src/main/kotlin/com/wire/crypto/CoreCrypto.kt
+
+    sed -i '' \
+        -e 's/[[:<:]]CryptoException[[:>:]]/CryptoError/g' \
+        -e 's/[[:<:]]E2eIdentityException[[:>:]]/E2eIdentityError/g' \
+        ./bindings/jvm/src/main/kotlin/uniffi/core_crypto/core_crypto.kt
+'''
```

Comment:
If the BSD sed supports `\<` and `\>` for word boundaries, you could modify the original sed invocation to be e.g.
```
       sed -i.bak \
        -e 's/\<CryptoException\>/CryptoError/g' \
        -e 's/\<E2eIdentityException\>/E2eIdentityError/g' \
        ./bindings/jvm/src/main/kotlin/uniffi/core_crypto/core_crypto.kt
```
The `\<` and `\>` are supported by GNU sed so we might be able to use the same command on both platforms.